src/sdltiles.cpp

```diff
@@ -3968,10 +3968,11 @@ std::optional<tripoint> input_context::get_coordinates( const catacurses::window
     }
 
     const point screen_pos = coordinate - win_min;
+    const bool use_isometric = g->w_overmap ? false : g->is_tileset_isometric();
```

Comment:
```suggestion
    const bool use_isometric = capture_win_ == g->w_overmap ? false : g->is_tileset_isometric();
```
Might be better to check against the argument.